projects/EditionsIgnite/app/theme/color.ts

```diff
@@ -1,68 +1,48 @@
 import { palette } from "@guardian/pasteup/palette"
 
-/**
- * Roles for colors.  Prefer using these over the palette.  It makes it easier
- * to change things.
- *
- * The only roles we need to place in here are the ones that span through the app.
- *
- * If you have a specific use-case, like a spinner color.  It makes more sense to
- * put that in the <Spinner /> component.
- */
+/*
+Roles for colors.  Prefer using these over the palette.  It makes it easier
+to change things. Also it makes night mode possible so there's that.
+
+The only roles we need to place in here are the ones that span through the app.
+If you have a specific use-case, like a spinner color.  It makes more sense to
+put that in the <Spinner /> component.
+*/
+
 export const color = {
-  /**
-   * The palette is available to use, but prefer using the name.
-   */
-  palette,
-  /**
-   * A helper for making something see-thru. Use sparingly as many layers of transparency
-   * can cause older Android devices to slow down due to the excessive compositing required
-   * by their under-powered GPUs.
-   */
-  transparent: "rgba(0, 0, 0, 0)",
-  /**
-   * The screen background.
-   */
+  /*
+  Backgrounds
+  */
   background: palette.neutral[100],
-  /**
-   * Text over the main tinting color.
-   */
+  dimBackground: palette.neutral[97],
+  darkBackground: palette.neutral[20],
+  textOverDarkBackground: palette.neutral[100],
+
+  /*
+  Brand (our blue)
+  */
   textOverPrimary: palette.neutral[100],
-  /**
-   * The main tinting color.
-   */
   primary: palette.brand.main,
-  /**
-   * The main tinting color, but darker.
-   */
   primaryDarker: palette.brand.dark,
-  /**
-   * A subtle color used for borders and lines.
-   */
+
+  /*
+  Border colors
+  */
```

Comment:
i like this comment style better than the starry vertical line and thats that on that

Reply:
Can we add this as a prettierc rule?